harbour-syncthing.desktop

```
@@ -11,3 +11,4 @@ Comment=Continuous Replication Synchronization Thing
 Permissions=Internet;AppLaunch
```

I wonder, maybe this would be a better fix?

```suggestion
Permissions=Internet;AppLaunch;Privileged
```

---

Yes, this works too. I have updated my PR accordingly